### PR TITLE
Ensure unicode string when operating over stderr

### PR DESCRIPTION
**Pull request by elyezer**
ssh stderr is now being converted to unicode string. Make sure to use
unicode strings when operating over stderr.

### CODE REVIEW

**Comment by Ichimonji10** [comment]
[ichimonji10@beech:robottelo]$ git grep result.stderr
    …
    tests/foreman/cli/test_user.py:        self.assertNotRegexpMatches(str(result.stderr), 'undefined method')
    …

Does the string above also need to be declared as unicode?

**Comment by Ichimonji10** [comment]
ACK pending question.

**Comment by elyezer** [comment]
@Ichimonji10 I think it is not necessary, but as it is already an string I have updated it.

**Comment by JacobCallahan** [comment]
ACK

**Comment by Ichimonji10** [comment]
Thank you @elyezer.

File: tests/foreman/cli/test_activationkey.py [hunk 1]
```diff
@@ -1698,7 +1698,7 @@ def test_negative_update_auto_attach(self):
         except CLIFactoryError as err:
             self.fail(err)
 
-        self.assertIn("'--auto-attach': value must be one of", result.stderr)
+        self.assertIn(u"'--auto-attach': value must be one of", result.stderr)
```
Comment (rplevka) [inline tests/foreman/cli/test_activationkey.py]: how about writing the string in a form:
```python
'\'--auto-attach\': value must be one of'
```
to follow our string-in-the-single-quotes rule?

Comment (Ichimonji10) [inline tests/foreman/cli/test_activationkey.py]: This seems like a great use case for double quotes, to me. Unlike a language like sh, there's no syntactic difference between single and double quotes. So when I see double quotes, I think "there must be some specific reason double quotes are used." And so it is.

Comment (rplevka) [inline tests/foreman/cli/test_activationkey.py]: @Ichimonji10: That is a perfectly valid argument and I take that :+1:

Comment (Ichimonji10) [inline tests/foreman/cli/test_activationkey.py]: :smile:

**Pull request merged**